### ISSUE TITLE
Home Inventory

### ISSUE THREAD

**Issue by Pertempto** [el-apps/.github#6]
## what is the problem?

1. Trying to find something and you don't remember which tote you put it in - so you have to dig through all of them until you find it.
2. Getting a bunch of supplies that you don't use often and then forgetting that you own them so you buy more.
3. Running out of BBQ sauce because it didn't get added to the shopping list when you started on the last bottle.

## what's on the market?

https://www.nerdwallet.com/article/insurance/home-inventory-app-template

## why make a custom app?

1. I want to gain experience with building inventory apps so that I can implement inventory apps for businesses in the future.
2. I want an inventory app that is highly optimized for our actual use cases.
3. I'm pretty sure most of those apps are more for covering possessions rather than consumable inventory like the stuff you keep in your pantry. I would like to have an app that has the flexibility to track pretty much everything in our house, if I'm willing to take the time to add it all
4. i want an app that easily supports taking a yearly or bi-yearly inventory check of our entire house - that way we can really become aware of all the unnecessary stuff we have and get rid of it

## how often would we use the custom app?

In our house, I would use this app multiple times a week. Maybe even daily. Organization is very important to me, I just need a good tool to help do it well.

##  would the investment of time spent in creating the app be worth the value produced by the app?

Very much. I just haven't had the time to dig into it yet. It's been one of my big goals for a while, I'm hoping to have some time to work on it this fall.

**Comment by Pertempto** [comment]
@DavidLandes I'm not sure exactly what you are imagining as far as actually developing the apps.

I was thinking we would probably work together to flesh out the design for each app, but the actual implementation might be more of an individual basis since we each have app ideas that we're excited about. We can also help add new features to each app as we need them, but I'm not sure it would be very sustainable to try to implement each new app as a team effort. It seems like it would require too much time coordination on a project that is supposed to be very relaxed time-wise. What do you think?

We will also need to work together to form some sort of common foundation for our apps, so that they integrate well. I'm thinking it might be useful to create a repo containing some base code that we can build each app off of, so that we don't have to spend a bunch of time on boilerplate for each new app.

**Comment by DavidLandes** [comment]
I like the thought of that. We should manage our own apps & what direction they go. But you could add an issue & I could approve it. That way we can help each other out without stepping on toes.

**Comment by Pertempto** [comment]
Yeah I think collaborating on design will be the important part. Otherwise we'll end up with apps that are too specific to one use case 😆

**Comment by Pertempto** [comment]
https://awesome-selfhosted.net/tags/inventory-management.html#
